APCAppCore/APCAppCore/Library/Categories/NewDataCollector/Collectors/APCHealthKitBackgroundDataCollector.m

```diff
@@ -90,7 +90,7 @@ - (void)start
         
         [self.healthStore requestAuthorizationToShareTypes:nil
                                                  readTypes:readTypes
-                                                completion:nil];
+                                                completion:^(BOOL success __unused, NSError * _Nullable error __unused) {}];
```

Comment:
is this a swift-compatibility thing? for future reference... I often make use of nil completion handlers to keep the code clean, but it'd be nice to know if that's creating problems for swift devs down the road.